### ISSUE TITLE
TensorFlow.js Extension

### ISSUE THREAD

**Issue by playforge-coding** [OmniBlocks/monorepo#594]
How about running AI in the browser? The face sensing extension in Scratch and TW already use it. Not sure how you would load pre-trained models though...

@coderabbitai @OmniBlocks/boxy

**Comment by playforge-coding** [comment]
@coderabbitai @OmniBlocks/boxy Respond to this issue

**Comment by playforge-coding** [comment]
@OmniBlocks/boxy SNAP OUT OF IT

**Comment by playforge-coding** [comment]
> call:label_issue{label:enhancement}call:label_issue{label:question}

@OmniBlocks/boxy Your calls are not working SNAP OUT OF IT NOW

**Comment by supervoidcoder** [comment]
> How about running AI in the browser? The face sensing extension in Scratch and TW already use it. Not sure how you would load pre-trained models though...
> 
> [@coderabbitai](https://github.com/coderabbitai) [@OmniBlocks/boxy](https://github.com/orgs/OmniBlocks/teams/boxy)

like llms? 
oh easy just onnx runtime

except you need a lot of ram.... small models run decently with cpu, what really matters is ram so KABOOM EXPLODE

**Comment by playforge-coding** [comment]
@coderabbitai @OmniBlocks/boxy what about Transformers.js?

**Comment by playforge-coding** [comment]
@OmniBlocks/boxy it's basically just onnx runtime web but handles tokens for you. i have no idea why that would be worse than raw onnx.

**Comment by playforge-coding** [comment]
@supervoidcoder @OmniBlocks/boxy @coderabbitai it's ready!! see https://github.com/OmniBlocks/extensions/pull/27
